sections/main-collection-product-grid.liquid

```diff
@@ -104,16 +104,16 @@
                   <div class="facets__display">
                     <div class="facets__header">
                       {%- assign max_price_amount = filter.range_max | money | escape -%}
-                      <span class="facets__selected">{{ "sections.collection_template.max_price" | t: price: max_price_amount }}</span>
+                      <span class="facets__selected">{{ "sections.collection_template.max_price" | t: price: max_price_amount | strip_html }}</span>
```

Comment:
`strip_html` won't work on a `| escape`d value. We'd need to `strip_html` before escaping the text.

```suggestion
                      <span class="facets__selected">{{ "sections.collection_template.max_price" | t: price: max_price_amount }}</span>
```